libs/langchain/langchain/vectorstores/mongodb_atlas.py

```diff
@@ -145,7 +159,32 @@ def _insert_texts(self, texts: List[str], metadatas: List[Dict[str, Any]]) -> Li
         insert_result = self._collection.insert_many(to_insert)  # type: ignore
         return insert_result.inserted_ids
 
-    def _similarity_search_with_score(
+    def _similarity_search_query(
+        self,
+        query: dict[str, Any],
```

Comment:
Nit: I suggest using the capitalized versions of `Dict` and `List` throughout for consistency.

Reply:
Done, thanks!